### ISSUE TITLE
Is MESSAGE_RATELIMIT used at all??

### ISSUE THREAD

**Issue by okossuth** [Mailu/Mailu#1774]
Hello

I found that in the master and version 1.8 docs appears the MESSAGE_RATELIMIT setting but in the source code it doesnt seem to be used at all. Only appear in the docs subdir:
```
# grep -r  MESSAGE_RATELIMIT *
docs/compose/.env:MESSAGE_RATELIMIT=100/day
docs/configuration.rst:The ``MESSAGE_RATELIMIT`` is the limit of messages a single user can send. This is
```
So what is the current status of this setting? Can i use it in mailu 1.7 or 1.8 or is it a future addition in the works?

Thanks

**Comment by Diman0** [comment]
This issue can be closed as soon #1916 is merged.

**Comment by Diman0** [comment]
MESSAGE_RATELIMIT is now available on master (well technically in about ~20 minutes once the CI/CD workflow has finished). For details see #1916 
I'm closing this issue.
